### DISCUSSION TITLE
The Sun is what type of star?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#418] [Galaxy Trivia] [answered]
**Bonus:** What is the lifetime of this type of stars?

**Comment by kylekce** [comment, marked as answer]
The Sun is a G-type main-sequence star, also known as a yellow dwarf. The lifetime of this type of star is about 10 billion years.

**Comment by peterbaccaro** [comment]
The Sun is a 4.5 billion-year-old **yellow dwarf star** – a hot glowing ball of hydrogen and helium – at the center of our solar system.  Yellow dwarf stars live for about **10 billion years**, and at 4.5 billion years old, our middle-age sun is about halfway through its lifetime.
